doc/development/release_distribution.rst

```diff
@@ -15,137 +15,149 @@ Preamble
 This document covers the process for managing updates to the current minor release and making new releases.
 Within this document, the git remote ``upstream`` refers to the main QuTiP organsiation repository, and ``origin`` refers to your personal fork.
 
-Instructions on how to backport bugfixes to a release branch are detailed in bugfix_.
-You need to do this to make changes to a current release that cannot wait until the next minor release, but need to go out in a micro release as soon as possible.
+In short, the steps you need to take are:
 
-Follow either release_ if you are making a major or minor release, or microrelease_ instead if it is only a bugfix patch to a current release.
-For both, then do the following steps in order:
+1. Prepare the relevant ``git`` branch with the correct state of the code (git_).
+2. Build the binary and source versions of the package, and deploy it to PyPI (``pip``) (deploy_).
+3. Retrive the built documentation from GitHub (also in docbuild_).
+4. Release the files on the QuTiP GitHub page (github_).
+5. Update `qutip.org <http://qutip.org/>`_ with the new links and documentation (web_).
+6. Update the conda feedstock, deploying the package to ``conda`` (cforge_).
 
-1. docbuild_, to build the documentation
-2. deploy_, to build the binary and source versions of the package, and deploy it to PyPI (``pip``)
-3. github_, to release the files on the QuTiP GitHub page
-4. web_, to update `qutip.org <http://qutip.org/>`_ with the new links and documentation
-5. cforge_, to update the conda feedstock, deploying the package to ``conda``
 
 
-.. _gitwf:
+.. _git:
 
-Git workflow
-++++++++++++
+Setting Up The Release Branch
```

Comment:
I *think* one should not capitalize section headings, but I'm not enough of a grammar nazi to know for sure and I don't know if there is a QuTiP doc convention to follow.

Reply:
I'm pretty sure we don't have a convention.  Grammatically, title case (capitalise each word) is historically preferred (though you'd usually leave "the" without a capital), but in modern usage more and more people are just using sentence case.  I'm not especially bothered either way - I'm not even sure I'm consistent with myself day-to-day.